content/guides/tooling/typescript-support.md

```diff
@@ -152,19 +152,30 @@ instructions.
 
 ### Types for plugins
 
-You can utilize Cypress's type declarations in your
-[plugins file](/guides/tooling/plugins-guide) by annotating it like the
-following:
+You can utilize Cypress's type declarations by annotating the
+[Cypress configuration](/guides/references/configuration) like the following:
 
-```javascript
-// cypress/plugins/index.ts
+#### cypress.config.ts
 
+```javascript
 /// <reference types="cypress" />
```

Comment:
I'm not sure what this is - I don't think it's necessary.
```suggestion
```

Reply:
@cowboy this whole code example in the current docs is basically just to call out adding this line to utilize TS types. We can remove the whole thing if it's not needed anymore. 

https://docs.cypress.io/guides/tooling/typescript-support#Types-for-plugins